ruby/examples/app/controllers/user_registration_controller.rb

```diff
@@ -0,0 +1,31 @@
+class UserRegistrationController < ApplicationController
+  def new
+    @user = UserRegistration.new
+  end
+
+  def create
+    request = RegisterUser::Request.new(*user_params)
+    response = RegisterUser.new.register(request)
```

Comment:
This pattern is preferred over defaults in constructors, mainly because it is not the responsibility of those use case classes to know how to construct their concrete dependencies (even default ones!)